exchanges/stream/websocket_test.go

```diff
@@ -223,13 +223,17 @@ func TestConnectionMessageErrors(t *testing.T) {
 	assert.ErrorIs(t, err, errNoPendingConnections, "Connect should error correctly")
 
 	ws.useMultiConnectionManagement = true
+	ws.SetCanUseAuthenticatedEndpoints(true)
+	ws.verbose = true // NOTE: Intentional
```

Comment:
Why? Gotta say why in the comment. If its just trying to get coverage of a println I say remove. Its getting less than 1% coverage of the package for stuff that doesn't matter